ci-operator/step-registry/openshift/manifests/external/openshift-manifests-external-commands.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+LOCALPATH="${SHARED_DIR}/manifest_external.yaml"
+echo "${EXTERNAL_MANIFESTS_SHA256_HASH} -" > /tmp/sum.txt
+if ! curl -fLs "${EXTERNAL_MANIFESTS_URL}" | tee "${LOCALPATH}" | sha256sum -c /tmp/sum.txt >/dev/null 2>/dev/null; then
+  echo "Expected file at ${EXTERNAL_MANIFESTS_URL} to have checksum ${EXTERNAL_MANIFESTS_SHA256_HASH} but instead got $(< "${LOCALPATH}" sha256sum | cut -d' ' -f1)"
+  exit 1
+fi
+echo "Downloaded ${EXTERNAL_MANIFESTS_URL}, sha256 checksum matches ${EXTERNAL_MANIFESTS_SHA256_HASH}"
+
+# Check file syntax
+pip3 install pyyaml --user
+python3 -c 'import yaml
+import sys
+data = yaml.safe_load_all(open(sys.argv[1]))' "${LOCALPATH}"
```

Comment:
Installing PyYAML on the fly to confirm that some hash-confirmed download is still valid YAML seems a bit heavy, but I'm fine dropping this step later if it turns out to cause trouble due to PyPI outages or whatever.

Reply:
Ah, I guess we need PyYAML to shard out a multidoc manifest further down, so ignore this thread.

Reply:
Yeah, pyyaml is used in several places (and I can't believe its not in base python after all these years)